src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor().toString());
```

Comment:
```suggestion
        ball.setColor(new ColorSupplier().getRandomColor().name());
```

Reply:
I can't change it to  .name() . The error said that .toString() is redundant, but .name() doesn't find

Reply:
just remove toString() as you dont need this. getRandomColor() method returns you a String